instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -360,10 +360,10 @@ public void afterExecution(
 
   private void onSdkResponse(
       Span span, SdkResponse response, ExecutionAttributes executionAttributes) {
+    if (response instanceof AwsResponse) {
+      span.setAttribute("aws.request_id", ((AwsResponse) response).responseMetadata().requestId());
```

Comment:
could you add the `AttributeKey` for this at the start of the class with the `// copied from ...` comment